src/MoBi.UI/Services/ToolTipCreator.cs

```diff
@@ -65,8 +69,31 @@ private static SuperToolTip addFormulaToolTip(FormulaBuilderDTO formula, SuperTo
 
       public SuperToolTip ToolTipFor(UsedCalculationMethodDTO calculationMethod)
       {
-         var toolTip = CreateToolTip(calculationMethod.Description, calculationMethod.CalculationMethodName);
+         return CreateToolTip(calculationMethod.Description, calculationMethod.CalculationMethodName);
+      }
+
+      private SuperToolTip createCellValueToolTip(string cellValue)
+      {
+         if (cellValue == string.Empty)
+            return CreateToolTip();
+
```

Comment:
what's with the ...?

Reply:
It's the implementation of what is discussed in another comment.